framer/Utils.coffee

```diff
@@ -497,16 +504,7 @@ Utils.modulate = (value, rangeA, rangeB, limit=false) ->
 # STRING FUNCTIONS
 
 Utils.parseFunction = (str) ->
-
-	result = {name: "", args: []}
-
-	if _.endsWith str, ")"
-		result.name = str.split("(")[0]
-		result.args = str.split("(")[1].split(",").map (a) -> _.trim(_.trimEnd(a, ")"))
-	else
-		result.name = str
-
-	return result
+	Animator.parseFunction(str)
```

Comment:
Even though we internally only use it for Animation/Animator, I would not move this and let the Utils have as little dependencies as possible.